### DISCUSSION TITLE
Plano de Dados Abertos - Consulta e extração de dados

### DISCUSSION THREAD

**Discussion by jeffVolt** [BrasilAPI/BrasilAPI#250] [Show and tell]
Encontrei um link interessante que ajuda a entender o cenário brasileiro quanto aos dados abertos e como consultar os mesmos. É necessário se aprofundar um pouco mais, acredito que juntos vamos mais longe, sigo na investigação.

http://paineis.cgu.gov.br/QvAJAXZfc/opendoc.htm?document=ias%5Cprojeto%20ias.qvw&host=QVS%40idc-qlik-ias-p&anonymous=true&sheet=shPDA

**Comment by AlexandreUser** [comment]
Não sei se isso ajuda, mas tem uma API open source feita para mapear alguns dados publicos, e ela se mantem bem atualizada, se quiser ver mais aqui esta o link:
https://brasil.io/home

**Comment by AlexandreUser** [comment]
> Não sei se isso ajuda, mas tem uma API open source feita para mapear alguns dados publicos, e ela se mantem bem atualizada, se quiser ver mais aqui esta o link:
> https://brasil.io/home

Sem falar que essa API disponibiliza para download seu proprio dataset, ano passado, só de CNPJ baixei 82 GB

**Comment by jeffVolt** [comment]
Show de Bola Alexandre vejo assim.

**Comment by CarlosZiegler** [comment]
@AlexandreUser no que utilizou esses dados? Apenas por curiosidade mesmo.

**Comment by CarlosZiegler** [comment]
Sobre esse tema ainda, segue um website que tem um catálogo de API’s governamentais.
https://catalogo.conecta.gov.br/store/apis/list
mas acredito que algumas sao pagas.

**Comment by AlexandreUser** [comment]
> @AlexandreUser no que utilizou esses dados? Apenas por curiosidade mesmo.

Eu utilizei para uma API de KYC (Know Your Costumer)

**Comment by kibutzzz** [comment]
Também existe essa [API](https://servicodados.ibge.gov.br/api/docs) do IBGE que possui diversos endpoints que podem ser uteis.

**Comment by filipedeschamps** [comment]
Sensacional turma!!!!!!!

Gostaria de adicionar algo que também sempre foi um problema para mim, saber se um dia é útil no Brasil. No Pagar.me foi feita essa biblioteca: https://github.com/pagarme/business-calendar e lembroq que foi uma mão na roda!

Como andam esses dados @lucianopf ? Eles continuam sendo atualizados?

**Comment by lucianopf** [comment]
Mantemos atualizando sim @filipedeschamps , ainda é utilizado por vários sistemas da empresa!! :grimacing:

**Comment by ghost** [comment]
Boa tarde pessoal, dando meus centavos aqui sobre o assunto.. em se tratando de dados governamentais, se quisermos manter padrões de interoperabilidade entre datasets diferentes é interessante aderir ao eping, que é um padrão proposto pelo gov federal para os novos apps e para as manutenções evolutivas do legado.

se aderirmos a esse padrão, direta e indiretamente podemos sugerir melhorias nele e quiçá, gerar uma API genérica/abstrata o suficiente para ser usada para consumir outros dados abertos (DATASUS estou olhando pra você).

Abraços rapaziada! Sucesso é a única possibilidade!

http://eping.governoeletronico.gov.br/

**Comment by devarthurribeiro** [comment]
Falaaa @filipedeschamps  tem essa base com 3.738 conjuntos de dados abertos com podem ser consumidos por json:

http://www.dados.gov.br/dataset?res_format=JSON

**Comment by jonyw4** [comment]
> Sensacional turma!!!!!!!
> 
> Gostaria de adicionar algo que também sempre foi um problema para mim, saber se um dia é útil no Brasil. No Pagar.me foi feita essa biblioteca: https://github.com/pagarme/business-calendar e lembroq que foi uma mão na roda!
> 
> Como andam esses dados @lucianopf ? Eles continuam sendo atualizados?

@filipedeschamps eu tenho usado a API do [calendario.com.br](http://www.calendario.com.br/api_feriados_municipais_estaduais_nacionais.php) em um ecommerce à alguns anos para calcular a estimativa de entrega de um pedido. Não conheço muito a respeito, mas tem funcionado para mim. Pode ser uma interessante adição a discussão de vocês :+1:

**Comment by filipedeschamps** [comment]
Sensacional toda contribuição de todos 😍
